view/ViewableData.php

```diff
@@ -541,6 +542,9 @@ public function CSSClasses($stopAtClass = 'ViewableData') {
 			$classes[] = $this->template;
 		}
 
+		// Strip out namespaces
+		$classes = preg_replace('#.*\\\\#', '', $classes);
```

Comment:
CSS classes with slashes are invalid in any case. I can move the code, however, if it'll make it look better for the pull request.